### PR TITLE
Migration data

### PR DESCRIPTION
**Pull request by james-inglis-01**
We've created the migration files for people and messages. Also updated the README with instructions on how to run the migrations.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place your settings in this file to overwrite default and user settings.
```
Comment (don-smith) [inline .vscode/settings.json]: Please remove this file.

File: knexfile.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+// Update with your config settings.
+
+
+
+module.exports = {
```
Comment (don-smith) [inline knexfile.js]: Please remove all blank lines before this one.

File: knexfile.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+// Update with your config settings.
```
Comment (don-smith) [inline knexfile.js]: Please use 2 character indentation in this file instead of 4.

File: knexfile.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        pool: {
+            min: 2,
+            max: 10
+        },
+        migrations: {
+            tableName: 'knex_migrations'
+        }
+    }
+
+};
```
Comment (don-smith) [inline knexfile.js]: Remove semicolon.

File: migrations/20170815103308_people.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+exports.up = function(knex, Promise) {
+  return knex.schema.createTableIfNotExists('People', function(table) {
```
Comment (don-smith) [inline migrations/20170815103308_people.js]: `People` should be lowercase.

File: README.md [hunk 6]
```diff
@@ -10,3 +10,10 @@ After cloning this repo:
 npm i
 npm run dev
 ```
+## How to Run Migrations
```
Comment (don-smith) [inline README.md]: Please add a blank line before this header.

File: .vscode/settings.json (hunk 1 above)
Comment (james-inglis-01) [inline .vscode/settings.json]: Done

File: knexfile.js (hunk 3 above)
Comment (james-inglis-01) [inline knexfile.js]: Done

File: knexfile.js (hunk 2 above)
Comment (james-inglis-01) [inline knexfile.js]: Done

File: knexfile.js (hunk 4 above)
Comment (james-inglis-01) [inline knexfile.js]: Done

File: migrations/20170815103308_people.js (hunk 5 above)
Comment (james-inglis-01) [inline migrations/20170815103308_people.js]: Done

File: README.md (hunk 6 above)
Comment (james-inglis-01) [inline README.md]: Done

**Comment by james-inglis-01** [comment]
We added to the gitignore so that is why there is a conflict.

File: .vscode/settings.json (hunk 1 above)
Comment (don-smith) [inline .vscode/settings.json]: Are you sure? Did you do a `git rm .vscode/settings.json`?

**Pull request merged**